turbinia/workers/__init__.py

```diff
@@ -221,12 +222,15 @@ def close(self, task, success, status=None):
 
     if self.input_evidence:
       try:
-        self.input_evidence.postprocess()
+        self.input_evidence.postprocess(task_id=self.task_id)
       # Adding a broad exception here because we want to try post-processing
       # to clean things up even after other failures in the task, so this could
       # also fail.
       # pylint: disable=broad-except
       except Exception as exception:
+        with filelock.FileLock(config.RESOURCE_FILE_LOCK):
```

Comment:
Another super small thing, but maybe move this below the log line below just so when looking at the logs it's easier to understand why we are trying to remove the task from the resource a second time.